pkg/controller/migmigration/migmigration_controller.go

```diff
@@ -268,8 +273,9 @@ func (r *ReconcileMigMigration) postpone(migration *migapi.MigMigration) (time.D
 	migration.Status.SetCondition(migapi.Condition{
 		Type:     Postponed,
 		Status:   True,
+		Reason:   Postponed,
```

Comment:
we dont need this anymore, please drop this change

Reply:
Done, updated PR.